packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -105,12 +106,18 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     // invited users can autojoin, create their memberships in org
     if (autoJoinUsers.length) {
       await prisma.membership.createMany({
-        data: autoJoinUsers.map((userToAutoJoin) => ({
-          userId: userToAutoJoin.id,
-          teamId: team.id,
-          accepted: true,
-          role: input.role,
-        })),
+        data: autoJoinUsers.map((userToAutoJoin) => {
+          const organizationRole = userToAutoJoin?.teams?.[0]?.role;
+          return {
+            userId: userToAutoJoin.id,
+            teamId: team.id,
+            accepted: true,
+            role:
```

Comment:
this is to handle normal non pending members

[Screencast from 10-12-23 02:09:29 PM IST.webm](https://github.com/calcom/cal.com/assets/32706411/b2fe2c6a-2381-4199-8fa5-2c0d69d47101)